src/api/emit.js

```diff
@@ -10,11 +10,16 @@ const CustomEvent = ((Event) => {
 })(window.CustomEvent);
 
 function createCustomEvent(name, opts = {}) {
-  if (CustomEvent) {
-    return new CustomEvent(name, opts);
+  let e;
+  if (Event) {
+    e = new Event(name, opts);
```

Comment:
Far enough. I suppose I would have just added a `const { detail } = ops; delete opts.detail` to avoid passing `detail`. Passing random data into external APIs is generally a bad idea.